tests/tests/helm_lib_module_image_test.yaml

```diff
@@ -35,6 +35,25 @@ tests:
       testModule:
         registry:
           base: "deckhouse.io/external-modules"
+    asserts:
```

Comment:
That was probably missing in the main branch?

Reply:
probably